contrib/sly-mrepl.el

```diff
@@ -1025,13 +1025,15 @@ handle to distinguish the new buffer from the existing."
   (interactive (list (point) t))
   (let* ((point (or point (point)))
          (probe
-          (previous-single-property-change point
-                                           'sly-mrepl--prompt))
+           (previous-single-property-change point
+                                            'sly-mrepl--prompt))
          (package (and probe
                        (or (get-text-property probe 'sly-mrepl--prompt)
-                           (get-text-property (previous-single-property-change
-                                               probe 'sly-mrepl--prompt)
-                                              'sly-mrepl--prompt)))))
+                           (and (previous-single-property-change
```

Comment:
There's a duplicate call to `(previous-single-property-change probe 'sly-mrepl--prompt)` that should be factored out.

Reply:
fixed.